packages/fuel-gauge/src/experimental-logging.test.ts

```diff
@@ -113,108 +94,73 @@ describe('Experimental Logging', () => {
   it('logs boolean', async () => {
     const expected = true;
 
-    const { logs } = await contractInstance.functions
-      .log_boolean(expected)
-      .txParams({ gasPrice, gasLimit: 10_000 })
-      .call();
+    const { logs } = await contractInstance.functions.log_boolean(expected).call();
 
     expect(logs).toStrictEqual([expected]);
   });
 
   it('logs boolean boolean multiple params', async () => {
     const expected = [true, false];
 
-    const { logs } = await contractInstance.functions
-      .log_boolean_boolean(...expected)
-      .txParams({ gasPrice, gasLimit: 10_000 })
-      .call();
+    const { logs } = await contractInstance.functions.log_boolean_boolean(...expected).call();
 
     expect(logs).toEqual(expected);
   });
 
   it('logs number boolean mixed params', async () => {
     const expected = [U32_MAX, true];
 
-    const { logs } = await contractInstance.functions
-      .log_number_boolean(...expected)
-      .txParams({ gasPrice, gasLimit: 10_000 })
-      .call();
+    const { logs } = await contractInstance.functions.log_number_boolean(...expected).call();
 
     expect(logs).toEqual(expected);
   });
 
   it('logs b256', async () => {
     const expected = B256;
 
-    const { logs } = await contractInstance.functions
-      .log_b256(expected)
-      .txParams({ gasPrice, gasLimit: 10_000 })
-      .call();
+    const { logs } = await contractInstance.functions.log_b256(expected).call();
 
     expect(logs).toStrictEqual([expected]);
   });
 
   it('logs b512', async () => {
     const expected = B512;
 
-    const { logs } = await contractInstance.functions
-      .log_b512(expected)
-      .txParams({ gasPrice, gasLimit: 10_000 })
-      .call();
+    const { logs } = await contractInstance.functions.log_b512(expected).call();
 
     expect(logs).toStrictEqual([expected]);
   });
 
   it('logs b256 b512 multiple params', async () => {
     const expected = [B256, B512];
 
-    const { logs } = await contractInstance.functions
-      .log_b256_b512(...expected)
-      .txParams({ gasPrice, gasLimit: 10_000 })
-      .call();
+    const { logs } = await contractInstance.functions.log_b256_b512(...expected).call();
 
     expect(logs).toEqual(expected);
   });
 
   it('logs u8 vec', async () => {
     const expected = [U8_MAX, 1, U8_MAX, 5];
 
-    const { logs } = await contractInstance.functions
-      .log_vec_u8(expected)
-      .txParams({ gasPrice, gasLimit: 10_000 })
-      .call();
-
-    expect(logs).toEqual([expected]);
-  });
-
-  it('logs b256 vec', async () => {
```

Comment:
Did you mean to remove this test?

Reply:
Superseded it with the contract test. Can reintroduce if necessary but all logic is coverered by `experimental-contract`. Potentially it was removed by mistake when I was debugging contracts.

Reply:
These tests are testing the `logs` from the transaction, whereas `experimental-contract` is only testing the `value`. I would favour re-adding this test.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1832/commits/19740157f6244095905cf9a6c9fdec46f938bb21